patina_dxe_core/src/patina_test_support.rs

```diff
@@ -0,0 +1,279 @@
+//! DXE Core Patina Test Support
+//!
+//! Code to help support running Patina Tests.
+//!
+//! ## License
+//!
+//! Copyright (c) Microsoft Corporation.
+//!
+//! SPDX-License-Identifier: Apache-2.0
+//!
+
+mod stability_tests;
```

Comment:
This is a lot of code. I wonder if there is some way we can conditionally compile it only when patina_tests are enabled.